cypress/integration/canvas_test_spec.js

```diff
@@ -279,6 +279,7 @@ context('Test Canvas', function(){
                 canvas.getRightSideToolPalette().should('not.exist');
                 //add a canvas from Class work to rightside workspace
                 rightNav.openClassWorkTab();
+                rightNav.openClassWorkSections();
```

Comment:
@eireland Because we now have collapsable sections, Cypress tests must first open the sections before we can access the content within.